experimental/packages/opentelemetry-sdk-metrics-base/src/export/ConsoleMetricExporter.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ExportResult, ExportResultCode } from '@opentelemetry/core';
+import { InstrumentType } from '../InstrumentDescriptor';
+import { AggregationTemporality } from './AggregationTemporality';
+import { ResourceMetrics } from './MetricData';
+import { PushMetricExporter } from './MetricExporter';
+
+/* eslint-disable no-console */
+export class ConsoleMetricExporter implements PushMetricExporter {
+  protected _shutdown = false;
+
+  export(metrics: ResourceMetrics, resultCallback: (result: ExportResult) => void): void {
+    if (this._shutdown) {
+      setImmediate(resultCallback, { code: ExportResultCode.SUCCESS });
```

Comment:
If the exporter is shut down nothing will be displayed in the console, success result could be a little misleading here, any reason to do this way?

Reply:
As it the console metric exporter I don't think we have anything to dispose off?

Reply:
@pichlermarc  Would that mean that the InMemoryExporter is incorrect too?

Reply:
Most likely

Reply:
[The `InMemoryExporter` already does export `ExportResultCode.FAILURE`](https://github.com/open-telemetry/opentelemetry-js/blob/dcc9f860be9ecfb3b1cdcd08d8cfd197d5fa7962/experimental/packages/opentelemetry-sdk-metrics-base/src/export/InMemoryMetricExporter.ts#L43-L46), so it should be correct. :slightly_smiling_face:

Reply:
> [The `InMemoryExporter` already does return `ExportResultCode.FAILURE`](https://github.com/open-telemetry/opentelemetry-js/blob/dcc9f860be9ecfb3b1cdcd08d8cfd197d5fa7962/experimental/packages/opentelemetry-sdk-metrics-base/src/export/InMemoryMetricExporter.ts#L43-L46), so it should be correct. 🙂

And this is why you check :)